.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
Ignoring the package-lock, because this repo should always take latest master from `vets-website`.